src/Components/Steps/ZipcodeStep.tsx

```diff
@@ -141,7 +138,9 @@ export const ZipcodeStep = () => {
         />
         {shouldShowCountyInput && (
           <div>
-            <QuestionQuestion>{matchingQuestion.followUpQuestions[0].question}</QuestionQuestion>
+            {matchingQuestion.followUpQuestions && (
```

Comment:
There is always going to be a follow up question, so you don't need this conditional.

Reply:
Most questions don't have `followUpQuestions`. This also takes care of the `typescript` error.